src/qtiCommonRenderer/renderers/interactions/MediaInteraction.js

```diff
@@ -98,13 +98,8 @@ var render = function render(interaction) {
                     renderTo: $('.media-container', $container)
                 })
                     .on('render', function() {
-                        resize();
-
                         $(window)
-                            .off('resize.mediaInteraction')
-                            .on('resize.mediaInteraction', resize);
-
-                        $item.off('resize.gridEdit').on('resize.gridEdit', resize);
+                            .off('resize.mediaInteraction');
```

Comment:
Why do you need off, if you don't have on, and why have you moved this to window object and are you sure that object is still valid and it won't change component?

Reply:
I use off for ensure that events in this part are disabled, we have a lot of triggers in different parts of code

![image](https://user-images.githubusercontent.com/34692207/94801858-f2744680-03e6-11eb-8735-37ee32223755.png)

Reply:
@habemuscode I think @zagovorichev want you to remove this part of code since it does nothing:
```suggestion
         $(window).off('resize.mediaInteraction');
```
Also, I would suggest removing `resize` function since is not in used.
Are you relying on styles to change media dimensions on window resize?